### PR TITLE
python3: add -m venv example

### PR DESCRIPTION
**Pull request by linpan**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/python.md
Comment (EmilySeville7cfg) [inline pages/common/python.md:40]:
```suggestion
`python3 -m venv {{path/to/environment}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Comment by CleanMachine1** [comment]
Please sign the the CLA otherwise we cannot merge this PR.

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Is 3rd example very useful? I am not sure what code example to remove but it must be done to successfully pass linter checks.

**Review by CleanMachine1 — DISMISSED** [review]

**Comment by CleanMachine1** [comment]
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/git-terminal.md#changing-the-email-of-any-commits

this maybe useful for the CLA issue.

**Comment by linpan** [comment]
so long wait for winter

File: pages/common/python.md
Comment (EmilySeville7cfg) [inline pages/common/python.md:40]:
```suggestion
`python3 -m venv {{path/to/environment1 path/to/environment2 ...}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by marchersimon** [comment]
@linpan could you please make sure the CLA is fully signed and the page doesn't have more than 8 examples?

**Comment by EmilySeville7cfg** [comment]
Sorry, I just wanted these old PRs not to be closed and merged after at least two approvals.

**Comment by CleanMachine1** [comment]
I don't think there is much we can do when the CLA is not completely signed, even if the author has signed it on 1 email.

**Pull request closed without merging**
